src/authenticationservices.cs

```diff
@@ -608,6 +608,8 @@ interface ASAuthorizationSingleSignOnRequest {
 	enum ASAuthorizationAppleIdButtonType : long {
 		SignIn,
 		Continue,
+		[iOS (13, 2), Mac (10, 15), TV (13, 1)]
```

Comment:
The diff file shows `macos(10.15.1)` - do we add the `.1` here too?

Reply:
yes but https://github.com/xamarin/xamarin-macios/pull/7315
someone forgot to update/check the wiki ?

Reply:
sorry got merged by @mandel-macaque while I was out of the office. Updating now.

Reply:
Closing this PR 😆